contrib/packaging/docker/Dockerfile.runtime

```diff
@@ -10,15 +10,37 @@ apt-get upgrade -y && \
 apt-get install -y --no-install-recommends \
 # Additional iproute2 runtime dependencies
     libelf1 libmnl0 \
-# Additional BPF build runtime dependencies
-    libgcc-5-dev \
```

Comment:
We don't need this?

Reply:
libgcc-5-dev is not available in Ubuntu 20.04, but I was able to build the main cilium image without this, but maybe some other components will fail.

Reply:
This was apparently needed for clang (#514). Did you try building the BPF programs without the lib? Note that there seem to be later versions available on Ubuntu 20.04 if needed.

Reply:
@qmonnet the CI passed successful for this new image, do you think we still need it?

Reply:
20.04 has libgcc-7-dev available, should I bump to this version?

Reply:
I didn't realise CI had run with that change, sorry. All good for me then :)